### PR TITLE
Merge parse-zoneinfo into chrono-tz workspace

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

**Comment by pitdicker** [comment]
Do you plan to delete/archive the parse-zoneinfo repo?

**Comment by djc** [comment]
Yes, after replacing the README with a pointer to chrono-tz. Sound good?

File: .github/workflows/rust-test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Rust
```
Comment (pitdicker) [inline .github/workflows/rust-test.yml:1]: If you are looking for the cause of the CI errors... This file :smile:

**Review by pitdicker — APPROVED** [review]

**Comment by pitdicker** [comment]
Best to merge manually?

**Comment by djc** [comment]
Yeah, was doing that when I noticed that the merge commit did not have the current main as its parent (but an earlier commit), so doing another CI run to make sure I've recreated the merge commit correctly.

File: examples/asia [hunk 2]
```diff
@@ -0,0 +1,3651 @@
+# tzdb data for Asia and environs
```
Comment (pitdicker) [inline examples/asia:1]: Could this make use of the file in the `tz` submodule instead?

Comment (djc) [reply examples/asia:1]: That's a separate issue from merging the repos, let's do it in follow-up!

**Pull request merged**